common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -246,6 +247,23 @@ innovation_iron_star = {
 	flag = high_medieval_era_regional
 }
 
+innovation_storm_hammers = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	icon = @storm_hammers
+	potential = {
+		this = culture:wildhammer
+	}
+	maa_upgrade = {
+		type = light_aerial 			
+		damage = 10
+		pursuit = 5
```

Comment:
I wonder, you used the spreadsheet for this?

Reply:
a) we don't have a spreadsheet for techs afaik, at least not for techs improving army.
b) we'd need some balancing regarding how much stronger the tech should make them and how long the tech should take to research